src/Post/Post.js

```diff
@@ -10,6 +8,41 @@ import { useIGData } from '../hooks/useIGData';
 import { absoluteToRelativeDate } from '../utils';
 import './Post.css';
 
+function PostLikeButton({
+  is_post_liked,
+  index,
+  toggleLike,
+  commentId,
+  replyId,
```

Comment:
You can keep this because it's not that bad, but since it's a button, I would just add an `onClick` prop and handle the click in the parent instead. This way `PostLikeButton` won't need to know the `index`, the `toggleLike`, `commentId`, nor `replyId`.

```suggestion
  onClick,
```